docs/source/examples/2/environment.yml

```diff
@@ -3,4 +3,4 @@ dependencies:
   - pip=20.1.1
   - python=3.7.3
   - pip:
-    - hi-ml==0.1.3
+    - hi-ml-azure==0.1.3
```

Comment:
There will soon no longer be 0.1.3, can we make all example environments use >=0.1?